### DISCUSSION TITLE
Could debug-bridge plugin be used in the coming zotero 7?

### DISCUSSION THREAD

**Discussion by redleafnew** [retorquere/zotero-better-bibtex#2390] [General]
Could debug-bridge-6.7.9.emile.limonia.xpi  be used in the coming zotero 7? If not, will the excellent plugin update to adapt for zotero 7?

**Comment by retorquere** [comment]
The plugin will be adapted, but I'm running behind on the work of prepping BBT + debug bridge for 7.

**Comment by redleafnew** [comment]
Thanks.
It prompted me "The add-on "%S" could not be installed. It may be incompatible with this version of Zotero." now when I installed it in zotero 7.

**Reply by retorquere** [reply]
It doesn't currently work in Z7

**Comment by redleafnew** [comment]
OK. Looking forward the compatiable version.
